wrapper/wrap-template.js

```diff
@@ -415,7 +484,7 @@
     exports.memzero = memzero;
     exports.output_formats = output_formats;
     exports.symbols = symbols;
-    exports.to_base64 = to_base64;
```

Comment:
Thanks for catching that, that's unintentional it should have been an addition (I got it right when I edited dist/combined/sodium.js but not when I copied it back to wrap-template.js), and since I can't make on my Mac, I only tested the "dist" version.